tracing/config/tracing.go

```diff
@@ -67,7 +67,8 @@ func (cfg *Config) Equals(other *Config) bool {
 	return reflect.DeepEqual(other, cfg)
 }
 
-func defaultConfig() *Config {
+// NoopConfig returns a new noop default config
```

Comment:
```suggestion
// NoopConfig returns a new noop config
```
Or `default config`, but I think both are a bit too much :)